src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,94 @@
+
+## Miscellaneous Interactions
+
+- Hacking the criminal records computer is now one of the many miscellaneous actions the Space Ninja can do in order to frustrate the crew - The error announcement will now only be sent by the criminal records computer on the Security channel, opposed to the previous station-wide Central Command Announcement.
+- The crew monitoring server is now prone to being hacked by the Space Ninja, doing so will disable everyone's sensors until deconstructed and reconstructed.
+- The station records computer is now prone to being hacked by the Space Ninja, doing so will delete a small % of random personnel from it accompanied by a station-wide announcement.
+- The camera router is now prone to being hacked by the Space Ninja, doing so will kill a large % of its connected cameras and render them useless.
+
+# Balance Changes
+
+Changes mostly geared towards fixing previously outright broken mechanics, as well to accomodate the new additions.
+
+## Nerfs
+
+These nerfs are intended to make previous go-to combat tactics less prevalent, hence incentivizing a more varied playstyle.
+
+- [Fix ninja stunlocking](https://github.com/space-wizards/space-station-14/pull/33244)
+- All Space Ninjas are now bio-engineered humans, making it so they don't get specie-specific attributes such as tail-dragging.
+- Removes the extended-capacity survival box from the ninja's satchel, as there's no longer any use for it.
+- Lower the energy katana's one-handed damage.
+- Lower the ninja throwing star's piercing damage to compensate for the newly added ninja caltrops.
+- Lower the ninja throwing star's stamina damage.
+
+## Buffs
+
+Enhances some things on the Space Ninja's kit in order to make it less of a one-trick pony, as well as to fix some inconsistencies.
+
+- Space Ninjas no longer have a need to eat nor drink.
+- Space Ninjas now come outfitted with a ninja headset as per [Alwayswannahunt's PR](https://github.com/space-wizards/space-station-14/pull/32841) , green encryption key included.
+- The space ninja gloves can now unlock access-restricted lockers and crates.
+- The energy katana's slash rate is slightly faster than it was previously when one-handed to compensate for it's damage loss.
+- The ninja throwing stars can now be stacked, as well as thrown in spreads.
+- When phasing, the Space Ninja is able to discreetly steal from people.
+
+## Miscellaneous Tweaks
+
+Mostly new additions that weren't either complete buffs nor complete nerfs.
+
+- The katana dash is considerably quieter, at the cost of a longer recharge duration.
+- Grants the ability to wield the energy katana for a hard-hitting swing, at the cost of a slower slash rate. *When wielding the energy katana, the Space Ninja's reflect probability also increases.
```

Comment:
There's a lot of different things you could do with this:

Have the sword reflect an insane amount if the ninja keeps pointing at the attacker. Because, you know, the ninja can parry. The downside to this is that the ninja is exposed from all other sides: shooting the ninja is no longer just a gamble on whether you score a hit or not.

I think the reflect system could also be refactored to determine the reflect chance of lasers versus bullets, that way you could give the sword a much higher chance to reflect bullets as they travel slower than lasers that travel at the speed of light. This could also make balancing items that can reflect easier.

Reply:
> There's a lot of different things you could do with this:
> 
> Have the sword reflect an insane amount if the ninja keeps pointing at the attacker. Because, you know, the ninja can parry. The downside to this is that the ninja is exposed from all other sides: shooting the ninja is no longer just a gamble on whether you score a hit or not.

Interesting way to go about it, but it'd probably require some heavy changes to the reflect system in general to add things related to orientation. I'll think about it, however.

> I think the reflect system could also be refactored to determine the reflect chance of lasers versus bullets, that way you could give the sword a much higher chance to reflect bullets as they travel slower than lasers that travel at the speed of light. This could also make balancing items that can reflect easier.

Speaking of heavy changes, I agree with this wholeheartedly, lasers should be prone to reflection via stuff like appropriate vests or mirrors, and blades should also have an absorb chance when it comes to bullets, the latter which I think would work wonder for the ninja (basically cutting bullets in half)

Reply:
> Have the sword reflect an insane amount if the ninja keeps pointing at the attacker.

100% bullet reflect while moving max speed so you can pretend you're playing mgrr :trollface:

Reply:
> > Have the sword reflect an insane amount if the ninja keeps pointing at the attacker.
> 
> 100% bullet reflect while moving max speed so you can pretend you're playing mgrr :trollface:

.1% probability for the ninja to have one of jetstream sam's voicelines after a successful reflection ![image](https://github.com/user-attachments/assets/c43100f3-623f-42cc-b26b-08080cd7c036)